### PR TITLE
Remaining table issues

### LINKED ISSUE #177: Add filter to remove streams created by livepeer.org and livepeer.com users on the Admin streams page
https://livepeer.com/app/admin/streams

- Add a checkbox filter above the table next the "show active only" checkbox.
- This new checkbox should read "Exclude streams from a Livepeer email addresses" and this should only return users without @livepeer.com or @livepeer.org email addresses. 
- "Exclude users with Livepeer email address" should be unchecked by default.

@darkdarkdragon is also adding this filter to the users page as part of https://github.com/livepeer/livepeer-com/pull/99.

### LINKED ISSUE #178: Add a stream id column and search to the Admin streams page
https://livepeer.com/app/admin/streams

- Replace the Details and segments columns with a new stream `id` column. The column header should be "ID"

Also, instead of the first search box by `name`, please change that to search by `id`

### LINKED ISSUE #247: Default sort Admin user list by email
Like this: 

https://livepeer.com/app/admin

### LINKED ISSUE #248: Default sort Admin api keys list by last active
Like this: 

https://livepeer.com/app/admin/keys

### PR DESCRIPTION
**Pull request by julianbenegas**
**What does this pull request do? Explain your changes. (required)**

- Default sort Admin api keys list by last active
- Default sort Admin user list by email (asc)
- Add a stream id column and search to the Admin streams page
- Add filter to remove streams created by livepeer.org and livepeer.com users on the Admin streams page

**Specific updates (required)**

 - None significant

**How did you test each of these updates (required)**

- With the vercel preview link

**Does this pull request close any open issues?**

- closes #248 
- closes #247 
- closes #178 
- closes #177 

**Screenshots (optional):**

**Checklist:**

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/api/src/controllers/stream.js [hunk 1]
```diff
@@ -117,6 +118,10 @@ app.get("/", authMiddleware({ admin: true }), async (req, res) => {
   if (active && active !== "false") {
     query.push(sql`stream.data->>'isActive' = 'true'`);
   }
+  if (nonLivepeerOnly && nonLivepeerOnly !== "false") {
+    // TODO make this work!
+    // query.push(sql`stream.data->>'user.email' NOT LIKE '%livepeer%'`);
```
Comment (julianbenegas) [inline packages/api/src/controllers/stream.js]: @darkdarkdragon could you give me a hand with this query? My SQL knowledge is a bit limited.

Comment (darkdarkdragon) [reply packages/api/src/controllers/stream.js]: That means "return only objects in which 'email' field does not contain 'livepeer' in it"

Comment (darkdarkdragon) [reply packages/api/src/controllers/stream.js]: sorry, I'm wrong, it should be written other way. one sec.

Comment (julianbenegas) [reply packages/api/src/controllers/stream.js]: I think i solved it! [https://github.com/livepeer/livepeer-com/pull/251/files#diff-3075aa9b906db2f795522df0e1996840d19743fddd8c8c5d41cce6393efac594R122](https://github.com/livepeer/livepeer-com/pull/251/files#diff-3075aa9b906db2f795522df0e1996840d19743fddd8c8c5d41cce6393efac594R122)

**Comment by julianbenegas** [comment]
@wohlner the "Exclude streams from a Livepeer email address" is working for me on localhost, but not on https://livepeer-com-git-jb-remaining-table-issues.livepeerorg.vercel.app

Is there something about the  `/api` package that has a different preview workflow?

**Comment by darkdarkdragon** [comment]
@julianbenegas 
> but not on https://livepeer-com-git-jb-remaining-table-issues.livepeerorg.vercel.app

You've made changes to the /api part - that changes not reflected in xx.vercel.app deployments. You should merge it to master and wait till it will be deployed to livepeer.monster - then you can test them.

**Comment by julianbenegas** [comment]
> You should merge it to master and wait till it will be deployed to livepeer.monster - then you can test them.

Got it, thank you, @darkdarkdragon

**Comment by wohlner** [comment]
@julianbenegas can you please remove the `details` and `segments` columns from /app/admin/streams. It's mentioned on https://github.com/livepeer/livepeer-com/issues/178. You can keep the order id, user name, name, created at, last active, status.

**Comment by wohlner** [comment]
The exclude livepeer stream checkbox doesn't seem to work on https://livepeer-com-git-jb-remaining-table-issues.livepeerorg.vercel.app/app/admin/streams. Am I testing in the right spot?

**Comment by julianbenegas** [comment]
> @julianbenegas can you please remove the `details` and `segments` columns from /app/admin/streams. It's mentioned on #178. You can keep the order id, user name, name, created at, last active, status.

Gotcha

> The exclude livepeer stream checkbox doesn't seem to work on https://livepeer-com-git-jb-remaining-table-issues.livepeerorg.vercel.app/app/admin/streams. Am I testing in the right spot?

See [this comment](https://github.com/livepeer/livepeer-com/pull/251#issuecomment-772036509)

**Comment by adamsoffer** [comment]
> The exclude livepeer stream checkbox doesn't seem to work on https://livepeer-com-git-jb-remaining-table-issues.livepeerorg.vercel.app/app/admin/streams. Am I testing in the right spot?

@wohlner see Ivan's comment above ^

**Review by wohlner — APPROVED** [review]
My bad about the exclude livepeer checkbox. The table on /app/admin/streams looks good with the change you made. Let's get this deployed to livepeer.monster so we can test the exclude livepeer checkbox. cc: @adamsoffer

**Review by adamsoffer — APPROVED** [review]
Everything works great on my end. Great work. I'll merge so we can test #177.

**Comment by wohlner** [comment]
Works great on staging! if everyone else agrees, lets get this in prod @adamsoffer

**Pull request merged**